06-building-a-program-start-to-finish/solving-the-problem/output/invertedIndex.js

```diff
@@ -0,0 +1,193 @@
+/**
```

Comment:
You still have style violations. Are you aware there's an 80 character per line limit?